docs/CONFIG-VARS.md

```diff
@@ -238,8 +238,8 @@ stateful = {
 
 | Name | Description | Type | Default | Notes |
 | :--- | ---: | ---: | ---: | ---: |
-| enable_registry_access | Enable access from the GKE Cluster to the GCR for your Google Project | bool | true | adds the "Artifact Registry Reader" and
-"Storage Object Viewer" Roles to the Service Account associated with the Node VMs. |
+| enable_registry_access | Enable access from the GKE Cluster to the GCR for your Google Project | bool | true | adds the "Artifact Registry Reader" and "Storage Object Viewer" Roles to the Service Account associated with the Node VMs. |
+
```

Comment:
Please clarify what changed here.

Reply:
Removed a newline between the words and & "Storage
That extra newline was causing the note to be split into multiple cells, You can see that here in the main branch: https://github.com/sassoftware/viya4-iac-gcp/blob/main/docs/CONFIG-VARS.md#google-container-registry-gcr